HISTORY.markdown

```diff
@@ -7,7 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+* Fixed issue [#500][github-issue-500] with grecaptcha2. See [#502][github-pr-502].
```

Comment:
I think in changelog we need to talk about `Cannot read property 'type' of null` and messages, not about grecaptcha2. Cause this error can be in other cases too.